src/Land/Model/SoilHeatParameterizations.jl

```diff
@@ -184,23 +188,108 @@ end
 
 """
     volumetric_internal_energy_liq(
-        cp_l::FT,
         T::FT,
         T_ref::FT,
     ) where {FT}
+
 Compute the expression for the volumetric internal energy of liquid water.
-Here, cp_l is the volumetric heat capacity of liquid water.
 """
 function volumetric_internal_energy_liq(
     T::FT,
     param_set::AbstractParameterSet,
 ) where {FT}
-
     _T_ref = FT(T_0(param_set))
     _ρ_l = FT(ρ_cloud_liq(param_set))
     ρcp_l = FT(cp_l(param_set) * _ρ_l)
     ρe_int_l = ρcp_l * (T - _T_ref)
     return ρe_int_l
 end
 
+"""
+    function k_solid(
+        ν_ss_om::FT,
+        ν_ss_quartz::FT,
+        ν_ss_minerals::FT,
```

Comment:
removed this line, will push after the current CI passes (want to make sure it passes/docs look good)